tic-tac-toe-test/RulesTest.cs

```diff
@@ -0,0 +1,79 @@
+using Xunit;
+using tic_tac_toe;
+using System.Collections.Generic;
+
+namespace tic_tac_toe_test
```

Comment:
Is understores in namespace idiomatic? I'm not sure but this makes me think it should be PascalCase https://csharp.2000things.com/2012/03/02/531-best-practices-for-naming-namespaces/

Reply:
Yeah... you might be right... I will change it.

I was following [this blog post](https://softchris.github.io/pages/dotnet-core.html#creating-a-test-library) to learn how to get started with a .NET project  and the author is using snake case... but I think that might be a bit odd. The official docs for the "using directive" are also mentioning PascalCase...
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive